application/src/main/java/org/opentripplanner/standalone/config/buildconfig/TransferParametersMapper.java

```diff
@@ -0,0 +1,44 @@
+package org.opentripplanner.standalone.config.buildconfig;
+
+import static org.opentripplanner.standalone.config.framework.json.OtpVersion.V2_7;
+
+import org.opentripplanner.graph_builder.module.TransferParameters;
+import org.opentripplanner.standalone.config.framework.json.NodeAdapter;
+
+public class TransferParametersMapper {
```

Comment:
These parameters don't show up in the `BuildConfiguration.md` now. If I remember correctly, you should at least add them to https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/application/src/test/resources/standalone/config/build-config.json and https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/application/src/test/java/org/opentripplanner/generate/doc/framework/NodeAdapterHelper.java#L8-L20 as well.

Reply:
I added changes to both files